astropy/modeling/fitting.py

```diff
@@ -1156,6 +1154,62 @@ def _convert_input(x, y, z=None, n_models=1, model_set_axis=0):
     return farg
 
 
+class MinimizeFitter(Fitter):
```

Comment:
Regarding the name of the class - all other fitter names follow the pattern of  `<OptimizerName><StatisticName>Fitter`.
This fitter uses `leastsquare` statistic with no option to change it. Following the naming convention should the name be `MinimizeLSQFitter`?